### PR TITLE
Update dependencies and fix out of the box testing

### PR DESCRIPTION
**Pull request by cbetta**
Resolves parts of #48 and #47.

* Update tap to ^14.6.1 in order to resolve many security issues
* Update log4js to ^5.0.0 in order to resolve more security issues
* Explicitly use the en_US locale in tests
* Add the `ilib-loctool-javascript` dev dependency to allow out of the
  box testing to work

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,13 +76,14 @@
...
         "remark-stringify": "^6.0.4",
-        "tap": "^7.0.0",
+        "tap": "^14.6.1",
         "unified": "^7.1.0",
         "unist-util-filter": "^1.0.0",
         "unist-util-map": "^1.0.4",
         "xml2json": "^0.11.2"
     },
     "devDependencies": {
+        "ilib-loctool-javascript": "^1.0.0",
```
[the remark this answers is not in the document]
Comment (cbetta) [reply package.json]: Well in that case we need to change the README.md.

Just to clarify, when I clone this repo, install the dependencies, and then run `npm test` I get the following error:

```sh
$ npm test

> loctool@2.2.1 test /Users/cbetta/loctool
> cd test; node testSuite.js

Error: Cannot find module 'ilib-loctool-javascript'
Require stack:
- /Users/cbetta/loctool/test/testCustomProject.js
- /Users/cbetta/loctool/test/testSuite.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:625:15)
    at Function.Module._load (internal/modules/cjs/loader.js:527:27)
    at Module.require (internal/modules/cjs/loader.js:683:19)
    at require (internal/modules/cjs/helpers.js:16:16)
    at Object.<anonymous> (/Users/cbetta/loctool/test/testCustomProject.js:25:30)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:643:32)
    at Function.Module._load (internal/modules/cjs/loader.js:556:12)
    at Module.require (internal/modules/cjs/loader.js:683:19)
npm ERR! Test failed.  See above for more details.
```

[the remark this answers is not in the document]
Comment (cbetta) [reply package.json]: FWIW I think this is where it's failing: https://github.com/iLib-js/loctool/blob/development/test/testCustomProject.js#L25

File: package.json [hunk 2]
```diff
@@ -52,7 +52,7 @@
         "node": ">=6.0.0"
     },
     "scripts": {
-        "test": "cd test; node testSuite.js",
+        "test": "cd test; LANG=en_US node testSuite.js",
```
[the remark this answers is not in the document]
Comment (cbetta) [reply package.json]: Fixed

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (cbetta) [reply package.json]: Removed.

**Comment by cbetta** [comment]
🎉

**Comment by ehoogerbeets** [comment]
Node 6 does not support log4js 5, so I made my own PR with log4js 4 instead:

https://github.com/iLib-js/loctool/pull/63

**Pull request closed without merging**